tests.txt

```diff
@@ -303,6 +303,7 @@ aws-eks/test/integ.eks-cluster-private-endpoint.js
 aws-eks/test/integ.eks-cluster.js
 aws-eks/test/integ.eks-helm-asset.js
 aws-eks/test/integ.eks-inference.js
+aws-eks/test/integ.eks-inference-nodegroup.js
```

Comment:
```suggestion

```
Do not think we need to update this file.

Reply:
afaik, the integration tests were tying to run the `.ts` file in that case and failing. I can try again.